Control/move_to_pose/move_to_pose.py

```diff
@@ -173,16 +172,14 @@ def transformation_matrix(x, y, theta):
 def main():
 
     for i in range(5):
-        x_start = 20 * random()
-        y_start = 20 * random()
-        theta_start = 2 * np.pi * random() - np.pi
+        x_start: float = 20 * random()
+        y_start: float = 20 * random()
+        theta_start: float = 2 * np.pi * random() - np.pi
```

Comment:
Why type annotations are needed? How about just change the "20" to "20.0"?

Reply:
Yup, will do the change.